app/services/taxonomy_parser.rb

```diff
@@ -0,0 +1,14 @@
+class TaxonomyParser
+  def self.parse(content_item)
+    return [] unless content_item.key?(:links)
+
+    links = content_item.fetch(:links).deep_symbolize_keys
```

Comment:
Implemented suggestions in 69258ba8dd7ac01cc40cce9278f74decd69630f0. The failing test was fixed by updating the tests in `spec/services/content_items_service_spec.rb` to stub a hash with a taxons key rather than a symbol - the service now assumes each content item is a hash and does a bit more than just returning it up the call chain.